kubernetes/compose/v1beta1/scale.go

```diff
@@ -0,0 +1,77 @@
+package v1beta1 // import "github.com/docker/cli/kubernetes/compose/v1beta1"
+
+import (
+	"errors"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+// Scale scale a service to the specified number of replicas
```

Comment:
v1beta1 didn't support scale, it should still not support it (i.e. it comes in v1beta2).. so this should not be required